### DISCUSSION TITLE
cors headers not set for Edge browser when accessing Userinfo end point

### DISCUSSION THREAD

**Discussion by shayannyc25** [OpenIDC/mod_auth_openidc#901] [General]
hi
we are using apahce mod auth oidc to protect a site and also we have configured our apache web server to set some cors related headers. When we access the userinfo url exposed by mod auth oidc  in Chrome in XHR call, we get the cors headers but we access the same user info url in Edge, we dont get cors headers. Our apache web server config file is configured to set these headers for all browsers/user agents. Below are the cors headers that we have configured to be set  in the apache web server config file.
![image](https://user-images.githubusercontent.com/11200259/184385444-d2947ec6-6d4f-44a3-b811-3ee9fcb530f7.png)

**Comment by zandbelt** [comment]
that seems unrelated to mod_auth_openidc as far as I can tell
